packages/client-sdk-nodejs/package.json

```diff
@@ -18,7 +18,8 @@
     "integration-test-auth": "jest auth-client.test.ts --maxWorkers 1",
     "integration-test-vector": "jest vector-control-plane.test.ts vector-data-plane.test.ts --maxWorkers 1",
     "unit-test": "jest unit",
-    "integration-test": "jest integration --testPathIgnorePatterns \"auth-client.test.ts|vector-control-plane.test.ts|vector-data-plane.test.ts\" --maxWorkers 1",
+    "integration-test-leaderboard": "jest leaderboard --maxWorkers 1",
+    "integration-test": "jest integration --testPathIgnorePatterns \"auth-client.test.ts|vector-control-plane.test.ts|vector-data-plane.test.ts|leaderboard.test.ts\" --maxWorkers 1",
```

Comment:
@anitarua I think if we still want the leaderboard tests to run in github CI then we will have to update one or two of those github yaml files too?

Reply:
oh yeah you're right, I'll add that